packages/react/src/components/Tooltip/index.tsx

```diff
@@ -78,13 +77,28 @@ export default function Tooltip({
     }
   );
 
-  // Show the tooltip
-  const show: EventListener = useCallback(async () => {
-    // Clear the hide timeout if there was one pending
+  const hideTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+  const maybeSetHideTimeout = (fn: () => void, delay: number) => {
```

Comment:
What's this change for?

Reply:
This was part of some cleanup for a dangling timeout that was causing a memory leak, but it's no longer relevant